components/automate-ui/src/styles.scss

```diff
@@ -342,6 +342,29 @@ input {
   top: -2px;
 }
 
+.chef-servers-icon {
+  display: inline-block;
+  border: 1px solid $chef-darkest;
+  border-radius: 2px;
+  height: 16px;
+  width: 16px;
+  background-color: var(--chef-white);
+  position: relative;
+  top: 2px;
+  margin-right: 5px;
+}
+
+.chef-servers-icon:before {
+  color: $chef-darkest;
+  content: 'C';
+  font-size: 9px;
+  font-weight: bolder;
+  position: absolute;
+  right: 4px;
+  text-align: center;
+  top: -3px;
```

Comment:
[nit] @susanev / @jonong1972 / @apriofrost  - not sure who this falls under.

When I spin this up - it looks off center to me and `top` should be at `-4px` maybe?

Reply:
![custom-c-4](https://user-images.githubusercontent.com/54940695/81983021-52dd3400-9650-11ea-9064-c26699fed02a.png)

Reply:
yah, noticed this when the waivers one was added as well. ideally id love for them to be centered, but if that isn't easy thats okay too.

Reply:
@sachin-msys - there must be some browser differences in play.  I get the opposite of what you see with those settings.  🤷‍♂️   

No need for this to be a blocker.  I'll toy with it in my spare time soon.